02-data-preparation.ipynb

```diff
@@ -1337,298 +1334,17 @@
-       "      <td>0</td>\n",
-       "      <td>16</td>\n",
-       "      <td>BL</td>\n",
-       "      <td>TARAVAL</td>\n",
-       "      <td>37.733950</td>\n",
-       "      <td>-122.489714</td>\n",
-       "    </tr>\n",
-       "  </tbody>\n",
-       "</table>\n",
-       "<p>884262 rows × 15 columns</p>\n",
-       "</div>"
-      ],
-      "text/plain": [
-       "            Id  DayOfWeek  Day  Month  Year  Hour  Minute  Season  Night  \\\n",
-       "0            0     Sunday   10      5    12    23      59  Spring      1   \n",
-       "1            1     Sunday   10      5    12    23      51  Spring      1   \n",
-       "2            2     Sunday   10      5    12    23      50  Spring      1   \n",
-       "3            3     Sunday   10      5    12    23      45  Spring      1   \n",
-       "4            4     Sunday   10      5    12    23      45  Spring      1   \n",
-       "...        ...        ...  ...    ...   ...   ...     ...     ...    ...   \n",
-       "884257  884257  Wednesday    1      1     0     0       1  Winter      1   \n",
-       "884258  884258  Wednesday    1      1     0     0       1  Winter      1   \n",
-       "884259  884259  Wednesday    1      1     0     0       1  Winter      1   \n",
-       "884260  884260  Wednesday    1      1     0     0       1  Winter      1   \n",
-       "884261  884261  Wednesday    1      1     0     0       1  Winter      1   \n",
-       "\n",
-       "        Holiday  Block StreetType PdDistrict        lat        long  \n",
-       "0             0     21         AV    BAYVIEW  37.735051 -122.399588  \n",
-       "1             0      0        INT    BAYVIEW  37.732432 -122.391523  \n",
-       "2             0     21         ST   NORTHERN  37.792212 -122.426002  \n",
-       "3             0     48         ST  INGLESIDE  37.721412 -122.437394  \n",
-       "4             0     48         ST  INGLESIDE  37.721412 -122.437394  \n",
-       "...         ...    ...        ...        ...        ...         ...  \n",
-       "884257        0     27         ST    MISSION  37.751987 -122.408983  \n",
-       "884258        0     20         ST   NORTHERN  37.792681 -122.425342  \n",
-       "884259        0     56         ST  INGLESIDE  37.712075 -122.445418  \n",
-       "884260        0     16         AV    BAYVIEW  37.739479 -122.387394  \n",
-       "884261        0     16         BL    TARAVAL  37.733950 -122.489714  \n",
-       "\n",
-       "[884262 rows x 15 columns]"
-      ]
-     },
-     "execution_count": 196,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
    "source": [
     "encoded_train = train.copy() \n",
-    "encoded_test = test.copy()\n",
-    "\n",
-    "encoded_train['Year'] = encoded_train['Year'] - (encoded_train['Year'].min())\n",
-    "encoded_test['Year'] = encoded_test['Year'] - (encoded_test['Year'].min())\n",
-    "encoded_test"
+    "encoded_test = test.copy()"
```

Comment:
Removed out the following lines, because Year is not present anymore at this stage.